packages/blockchain-wallet-v4-frontend/src/modals/Nfts/components/NftDropdown.tsx

```diff
@@ -61,14 +67,19 @@ const FeesDropdown: React.FC<Props> = ({ children, totalFees }) => {
         </Flex>
       </Top>
       <FeesWrapper style={isActive ? {} : { display: 'none' }}>
-        {React.Children.map(children, (child) => (child ? <FeeChild>{child}</FeeChild> : null))}
+        {React.Children.map(children, (child) =>
+          child ? <FeeChild hasPadding={hasPadding}>{child}</FeeChild> : null
+        )}
       </FeesWrapper>
     </Wrapper>
   )
 }
 
 type Props = {
-  totalFees: string
+  expanded?: boolean
+  hasPadding?: boolean
+  title?: string
+  totalFees?: string
 }
 
-export default FeesDropdown
+export default NftDropdown
```

Comment:
If you're gonna do this gotta make it totally generic

1. change totalFees to titleRight
2. change FeeChild to Child
3. change FeesWrapper to ChildWrapper

What makes more sense to me tho is having an NftDropdown that then can be used by FeesDropdown